.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
   - . vendor/ci/travis/static-analysis.sh
 
 script:
-  - phpunit $PHPUNIT_FLAGS
+  - php vendor/bin/phpunit $PHPUNIT_FLAGS
```

Comment:
I had never issues with PHPUnit version. When it's with Prophecy (PHPUnit 4.7), it's fine. And on Travis it's quite up to date.